src/examples/bad_code.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+
+double calc_pi    (void)
+{
+    ROS_INFO("calculating pi...");
+        double result;
```

Comment:
Same complaint here with adding an extra tab to double result. Adding an extra tab should usually only happen when something is going to be nested inside the last line that was not indented.